lib/operations/command.js

```diff
@@ -36,6 +36,16 @@ class CommandOperation extends OperationBase {
   constructor(db, command, options, collection) {
     super(options);
 
+    if (!this.hasAspect(Aspect.WRITE_OPERATION)) {
```

Comment:
We should, either in `OperationBase` or here, be resolving `WriteConcern`, `ReadConcern`, and `retryWrites`